app/components/CapacityBar.tsx

```diff
@@ -60,16 +65,16 @@ export const CapacityBar = ({
       <div>
         <div className="flex justify-between border-t border-secondary">
           <div className="p-3 text-mono-sm">
-            <div className="text-quaternary">Provisioned</div>
+            <div className="text-quaternary">{provisionedLabel}</div>
             <div className="text-secondary">
-              {provisioned.toLocaleString()}
+              {displayBigNum(provisioned)}
               <span className="normal-case">{includeUnit ? ' ' + unit : ''}</span>
             </div>
           </div>
           <div className="p-3 text-mono-sm">
-            <div className="text-quaternary">{allocatedLabel}</div>
-            <div className="text-secondary">
-              {allocated.toLocaleString()}
+            <div className="text-quaternary">{capacityLabel}</div>
+            <div className="!normal-case text-secondary">
+              {displayBigNum(capacity)}
```

Comment:
This component is getting out of hand. I will make an issue about breaking it up into composable components.